modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/ErrorAlert/ErrorAlert.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable no-unused-vars */
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayAlert from '@clayui/alert';
+import ClayLink from '@clayui/link';
+import React, {useContext, useState} from 'react';
+
+import {
+	GPS_API_KEY_ERROR_CODE,
+	GPS_API_KEY_INVALID_STATUS,
+	GPS_SERVER_ERROR_CODE,
+} from '../../constants/pageSpeed';
+import {StoreStateContext} from '../../context/StoreContext';
+import ErrorAlertExtendedInfo from './ErrorAlertExtendedInfo';
+
+const ErrorAlert = () => {
+	const [showErrorInfo, setShowErrorInfo] = useState(false);
+	const expandInfoHandler = () =>
+		setShowErrorInfo((currentValue) => !currentValue);
+
+	const {data, error} = useContext(StoreStateContext);
+
+	const {configureGooglePageSpeedURL, privateLayout} = data || {};
+	const {code: statusCode, status} = error || {};
+
+	const apiKeyError =
+		statusCode === GPS_API_KEY_ERROR_CODE &&
+		status === GPS_API_KEY_INVALID_STATUS;
+
+	const serverError = statusCode === GPS_SERVER_ERROR_CODE;
+	const pageCanNotBeAudited = serverError || privateLayout;
+
+	const unknownError = !apiKeyError && !pageCanNotBeAudited;
+
+	const title = apiKeyError
+		? Liferay.Language.get('incorrect-api-key')
+		: pageCanNotBeAudited
+		? Liferay.Language.get("this-page-can't-be-audited")
+		: Liferay.Language.get('an-unexpected-error-occurred');
+
+	const actionTitle = showErrorInfo
+		? Liferay.Language.get('hide-error-details')
+		: Liferay.Language.get('show-error-details');
+
+	return (
+		<ClayAlert
+			displayType={apiKeyError || unknownError ? 'danger' : 'warning'}
+			variant="stripe"
+		>
+			<p>
+				<span
+					className={
+						pageCanNotBeAudited ? 'font-weight-bold' : undefined
+					}
+				>
+					{title}{' '}
+				</span>
+				{pageCanNotBeAudited &&
+					Liferay.Language.get(
+						'page-not-accessible-from-internet'
+					)}{' '}
+				{(apiKeyError || serverError) && (
+					<strong
```

Comment:
This element can not be selected with keyboard navigation right now. Maybe we could use another tag like 'a' (this way we don't need the cursor-pointer class) or we could add a tabindex.

Reply:
Thank you Clara! Much better with Clay button